setup.sh

```diff
@@ -103,694 +144,259 @@ vercomp () {
+  echo
 }
 
-installSystemService() {
-  KUZZLE_OPT_PATH="/opt/kuzzle"
-  SYSTEMD_FILE_PATH="/lib/systemd/system/kuzzle.service"
-  if [[ `systemctl` =~ -\.mount ]]; then
+pull_kuzzle() {
+  write_info "[ℹ] Pulling latest version of Kuzzle from Dockerhub..."
+  $KUZZLE_PUSH_ANALYTICS'{"type": "pulling-latest-containers", "uid": "'$ANALYTICS_UUID'", "os": "'$OS'"}' $ANALYTICS_URL &> /dev/null
+  $(command -v docker-compose) -f $COMPOSE_YML_PATH pull
+  RET=$?
+  if [ $RET -ne 0 ]; then
+    $KUZZLE_PUSH_ANALYTICS'{"type": "pull-failed", "uid": "'$ANALYTICS_UUID'", "os": "'$OS'"}' $ANALYTICS_URL &> /dev/null
     echo
-    writeBold "[❓] Would you like to install Kuzzle as a systemd service? (y/N)"
-    if [ $EUID != 0 ]; then
-      SUDO_PREFIX="sudo"
-    fi
-    if [ ! -d $KUZZLE_OPT_PATH ]; then
-      systemServCmd_1="$SUDO_PREFIX mkdir $KUZZLE_OPT_PATH"
-    fi
-    systemServCmd_2="$SUDO_PREFIX cp $COMPOSER_YML_PATH $KUZZLE_OPT_PATH/docker-compose.yml"
-    systemServCmd_3="$SUDO_PREFIX systemctl enable $SYSTEMD_FILE_PATH"
-    write "   If you agree, I will execute the following commands"
-    if [ ! -z "$systemServCmd_1" ]; then
-      write "   $systemServCmd_1"
-    fi
-    write "   $systemServCmd_2"
-    write "   Then, I will create a systemd file in $SYSTEMD_FILE_PATH and enable it"
-    write "   $systemServCmd_3"
-    echo -n "> "
-    read systemdService trash
-      case "$systemdService" in
-        [yY])
-          if [ ! -z "$systemServCmd_1" ]; then
-            eval $systemServCmd_1
-          fi
-          eval $systemServCmd_2
-          SERVICE_FILE_CONTENT=$(cat << EndOfMessage
-[Unit]
-Description=Kuzzle
-After=docker.service
-Requires=docker.service
-
-[Service]
-WorkingDirectory=$KUZZLE_OPT_PATH
-ExecStartPre=-/usr/local/bin/docker-compose kill
-ExecStartPre=-/usr/local/bin/docker-compose rm -f
-ExecStartPre=-/usr/local/bin/docker-compose pull
-ExecStart=/usr/local/bin/docker-compose up
-ExecStop=/usr/local/bin/docker-compose kill
-
-[Install]
-WantedBy=multi-user.target
-EndOfMessage
-)
-          $SUDO_PREFIX tee "$SYSTEMD_FILE_PATH" <<< "$SERVICE_FILE_CONTENT" &> /dev/null
-          eval $systemServCmd_3
-          if [ $? -eq 0 ]; then
-            echo
-            writeBold "$GREEN" "[✔] Kuzzle service successfully installed."
-            write     "    Use the following commands to manage Kuzzle"
-            write     "    systemctl start|stop|status kuzzle"
-            echo
-            write     "[ℹ] You'll probably need root privileges to use them"
-          else
-            echo
-            writeBold "$RED" "[✖] Ooops! Kuzzle service installation failed."
-          fi
-        ;;
-        *)
-          writeBold "$BLUE" "Ok."
-        ;;
-      esac
+    write_error "[✖] Pull failed. Please ensure your docker is running."
+    write_error "    You can try and run docker with service docker start or dockerd."
+    write_error "    To know more please refer to https://docs.docker.com/config/daemon"
```

Comment:
Still trying to get rid of all the `please` in this script's messages, a proposal: `For more information, refer to ...`